source/_data/releases.cjs

```diff
@@ -0,0 +1,90 @@
+const { spawn: nodeSpawn } = require('node:child_process');
```

Comment:
I tried with `releases.ts` first, but 11ty didn't seem to execute my function. Would like to learn how to do it properly in TS

Reply:
Hm, I'm not sure we can do that until 11ty adds better support for TS and ESM.

Reply:
Oh, so that's expected! Good to know